// ngx-gauge/esm2020/gauge/gauge-directives.mjs
import { Directive } from "@angular/core";
import * as i0 from "@angular/core";
export class NgxGaugeAppend {
}
NgxGaugeAppend.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugeAppend, deps: [], target: i0.ɵɵFactoryTarget.Directive });
NgxGaugeAppend.ɵdir = i0.ɵɵngDeclareDirective({ minVersion: "12.0.0", version: "13.3.1", type: NgxGaugeAppend, selector: "ngx-gauge-append", exportAs: ["ngxGaugeAppend"], ngImport: i0 });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugeAppend, decorators: [{
            type: Directive,
            args: [{
                    selector: "ngx-gauge-append",
                    exportAs: "ngxGaugeAppend"
                }]
        }] });
export class NgxGaugePrepend {
}
NgxGaugePrepend.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugePrepend, deps: [], target: i0.ɵɵFactoryTarget.Directive });
NgxGaugePrepend.ɵdir = i0.ɵɵngDeclareDirective({ minVersion: "12.0.0", version: "13.3.1", type: NgxGaugePrepend, selector: "ngx-gauge-prepend", exportAs: ["ngxGaugePrepend"], ngImport: i0 });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugePrepend, decorators: [{
            type: Directive,
            args: [{
                    selector: "ngx-gauge-prepend",
                    exportAs: "ngxGaugePrepend"
                }]
        }] });
export class NgxGaugeValue {
}
NgxGaugeValue.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugeValue, deps: [], target: i0.ɵɵFactoryTarget.Directive });
NgxGaugeValue.ɵdir = i0.ɵɵngDeclareDirective({ minVersion: "12.0.0", version: "13.3.1", type: NgxGaugeValue, selector: "ngx-gauge-value", exportAs: ["ngxGaugeValue"], ngImport: i0 });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugeValue, decorators: [{
            type: Directive,
            args: [{
                    selector: "ngx-gauge-value",
                    exportAs: "ngxGaugeValue"
                }]
        }] });
export class NgxGaugeLabel {
}
NgxGaugeLabel.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugeLabel, deps: [], target: i0.ɵɵFactoryTarget.Directive });
NgxGaugeLabel.ɵdir = i0.ɵɵngDeclareDirective({ minVersion: "12.0.0", version: "13.3.1", type: NgxGaugeLabel, selector: "ngx-gauge-label", exportAs: ["ngxGaugeLabel"], ngImport: i0 });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.3.1", ngImport: i0, type: NgxGaugeLabel, decorators: [{
            type: Directive,
            args: [{
                    selector: "ngx-gauge-label",
                    exportAs: "ngxGaugeLabel"
                }]
        }] });
